xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -215,9 +256,23 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reads the next data item as a boolean value (major type 7).</summary>
+        <returns>The decoded <see cref="T:System.Boolean" /> value.</returns>
```

Comment:
You don't need to link to the return type, it already appears above of this description.
```suggestion
        <returns>The decoded value.</returns>
```